test/mongodb.test.js

```diff
@@ -462,7 +462,7 @@ describe('mongodb connector', function () {
 
             User.updateAll({}, {age: 40, '$set': {age: 39}},function(err,updatedusers) {
               should.not.exist(err);
-              updatedusers.should.be.equal(3);
+              updatedusers.count.should.be.equal(3);
```

Comment:
Consider using `updatedusers.should.have.property(count, 3)` instead, it will give you nicer failure messages. Another nice-to-have change: rename `updatedusers` to `info`. This comment applies to all tests you are changing in this patch, feel free to ignore it though.